transport/src/main/java/io/el/channel/DefaultChannelEventLoopGroup.java

```diff
@@ -0,0 +1,123 @@
+package io.el.channel;
+
+import io.el.concurrent.EventLoop;
+import io.el.concurrent.EventLoopChooserFactory;
+import io.el.concurrent.EventLoopChooserFactory.EventLoopChooser;
+import io.el.concurrent.Promise;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+public class DefaultChannelEventLoopGroup implements ChannelEventLoopGroup {
```

Comment:
Implementation of this class is not included in issue https://github.com/zeroFruit/el/pull/37